contracts/protocore/OriginProtocore.sol

```diff
@@ -130,4 +132,38 @@ contract OriginProtocore is MasterCopyNonUpgradable, GenesisOriginProtocore, Pro
             _targetBlockNumber
         );
     }
+
+
+    /* Public Functions */
+
+    /**
+     * @notice inForwardValidatorSet() function delegates the call to
```

Comment:
dont use "delegate" as it seems to imply `DELEGATE_CALL` as an opcode; and it's a `CALL`

"inForwardValidatorSet() function calls on SelfProtocore contract to query the forward validator set.

same for L156

Reply:
Done